logdata/constraints.lisp

```diff
@@ -89,7 +89,7 @@
 ;;                                     ;;
 ;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;
 
-(defun    (normalized-SIZE_LIMB)
+(defun    ((normalized-SIZE_LIMB :i5 :force))
```

Comment:
@letypequividelespoubelles I believe it should be impossible for `SIZE_LIMB - 1` to be negative when `LOGS_DATA != 0`, right?